### ISSUE TITLE
I use the command to generate as readme said, but it doesn't work 

### ISSUE THREAD

**Issue by a1324321** [tgulacsi/oracall#1]
i use like this
oracall -pattern 'PACK1.%' -package main -out pack1_functions.go -proto pack1.proto 'DB_USER/xxxxxx@192.168.0.12:1521/xxx'
the pack1 is my oracle pack name
![image](https://user-images.githubusercontent.com/5265881/57963506-9dea3b80-7957-11e9-812f-69b245a5d6e7.png)
but it doesn't work 
![image](https://user-images.githubusercontent.com/5265881/57963515-ca9e5300-7957-11e9-919c-730996101540.png)

**Comment by a1324321** [comment]
pls help,thanks a lot!

**Comment by tgulacsi** [comment]
Ok, I've fixed the README.
Try

    oracall -connect 'DB_USER/xxxxxx@192.168.0.12:1521/xxx' -base-dir ./pack -db-out ./pack/db/pack1 -pb-out ./pack/pb/pack1 'PACK1.%'

**Comment by a1324321** [comment]
it seens return an error log :
package=PACK1 msg=getSource error="SELECT text FROM user_source WHERE name = UPPER(:1) AND type = 'PACKAGE' ORDER BY line: context canceled"
panic: runtime error: slice bounds out of range
![image](https://user-images.githubusercontent.com/5265881/57966865-55984100-798a-11e9-9361-7e9e28c47ba5.png)
![image](https://user-images.githubusercontent.com/5265881/57966877-85dfdf80-798a-11e9-8a83-44ff7ff5eec8.png)

**Comment by tgulacsi** [comment]
It tries to read the source (`user_source`) view for comments.
Is it available? Can you `SELECT text from user_source where name like 'PACK1.%' AND type = 'PACKAGE'` ?

But maybe that's just the context canceled, and the real error is the "slice out of range".

What does the panic say?

A problem can be that your `t1` type is a REF CURSOR, and you didn't specify the returned columns - such as
```
  TYPE t1_rec IS RECORD (i PLS_INTEGER, s VARCHAR2(10), d DATE);
  TYPE t1 IS REF CURSOR RETURN t1_rec;
```

TL;DR; oracall needs a "strongly typed" ref cursor - see http://www.dba-oracle.com/plsql/t_plsql_cursor_variables.htm for example!

**Comment by a1324321** [comment]
I use 'PACK1' instead of 'PACK1.%' , then it return this . What does the error mean? Thanks a lot!
I does have the source file in path "github.com/gogo/protobuf/gogoproto/gogo.proto"
![image](https://user-images.githubusercontent.com/5265881/57991527-285ea680-7ae2-11e9-82d7-e03c795ea165.png)

**Comment by tgulacsi** [comment]
The errors are from the missing google/protobuf/timestamp.proto and github.com/gogo/protobuf/ogogproto/gogo.proto files. They should be under $GOPATH/src, or under the current directory.

Have the files (pack1.go, pack1.proto) appear ?
Then you may be able to compile the pack1.proto file with protoc - adjust the params in the printed error string.

A "go generate" may help to download the missing files, as the README says.

**Comment by a1324321** [comment]
this does generate successfully
![image](https://user-images.githubusercontent.com/5265881/58004127-0e8a8700-7b15-11e9-9016-3a69e83f1317.png)

and the pb.proto is this
![image](https://user-images.githubusercontent.com/5265881/58004034-c8cdbe80-7b14-11e9-8a76-d49a63166221.png)
![image](https://user-images.githubusercontent.com/5265881/58004183-38dc4480-7b15-11e9-9164-14e6ee95a91b.png)
but without my pack1 function in it , what shuould I do...
![image](https://user-images.githubusercontent.com/5265881/58004276-793bc280-7b15-11e9-8489-89d0e1f5ea41.png)
